ui/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Multi-stage build for LiteLLM UI (Dashboard + Docs)
+FROM node:18-alpine AS builder
+
+# Install dependencies and build docs
+WORKDIR /app/docs
+COPY docs/package*.json ./
+RUN npm ci --only=production
+COPY docs/ ./
+ENV DOCUSAURUS_BASE_URL=/docs/
+RUN npm run build
+
+# Install dependencies and build dashboard, to change API_URL in prod. 
+WORKDIR /app/dashboard
+COPY litellm-dashboard/package*.json ./
+RUN npm ci --only=production
+COPY litellm-dashboard/ ./
+ENV UI_BASE_PATH=/dashboard
+ARG API_URL=http://localhost:4000
+ENV API_URL=$API_URL
+RUN npm run build
```

Comment:
U can build Dashboard and Docs in 2 separate stages so that they can work independently i.e. if u change docs, dashboard won't be rebuit